src/containers/Secrets/Secrets.test.js

```diff
@@ -220,3 +238,37 @@ it('SecretsTable only renders tekton.dev annotations', () => {
 
   expect(queryByText(/badcontent/i)).toBeFalsy();
 });
+
+it('Secrets can be filtered on a single label filter', async () => {
```

Comment:
this test excludes a non-matching secret, this is good, but it would be great if we could also include a matching secret in the test data to verify fully that this works as expected